tools/symlink.js

```diff
@@ -0,0 +1,70 @@
+const fs = require("fs-extra");
+const path = require("path");
+const yargs = require("yargs");
+const { hideBin } = require("yargs/helpers");
+
+/**
+ * @typedef {{dataPath: string;symLinkName: string;}} FoundryConfig
+ */
+
+const argv = yargs(hideBin(process.argv))
+  .option("clean", {
+    alias: "c",
+    type: "boolean",
+    default: false,
+  })
+  .parseSync();
+
+/**
+ * @type {FoundryConfig}
+ */
+const foundryConfig = fs.readJSONSync(
+  path.resolve(__dirname, "..", "foundryconfig.json")
+);
+
+const linkDirectory = path.resolve(
+  foundryConfig.dataPath,
+  "Data",
+  "systems",
+  foundryConfig.symLinkName
+);
+
+if (argv.clean) {
+  handleProcess(removeSymlink());
+} else {
+  handleProcess(createSymlink());
+}
+
+async function createSymlink() {
+  console.log(`Linking dist to ${linkDirectory}.`);
+  const systemsDirectory = path.resolve(linkDirectory, "..");
+  if (!fs.pathExistsSync(systemsDirectory)) {
+    throw new Error(
+      `Link directory parent folder "${systemsDirectory}" not found.`
+    );
+  }
+
+  if (!fs.existsSync(linkDirectory)) {
+    // link path doesn't exist, create it.
+    fs.symlinkSync(path.resolve(__dirname, ".."), linkDirectory);
```

Comment:
this was originally awaited. Perhaps it's not necessary given the way the functions are now split up? If it works as is, then yeah async isn't necessary and we can ditch it from the function declaration

Reply:
I'll remove the need for `await` and `async` in the file all together by using the `*Sync` functions.